config/prod/prow/config_knative.yaml

```diff
@@ -620,6 +620,11 @@ presubmits:
   - build-tests: true
   - unit-tests: true
   - integration-tests: false
+  knative-sandbox/container-freezer:
+  - build-tests: true
+  - unit-tests: true
+  - integration-tests: true
+  - go-coverage: false
```

Comment:
not that Im personally a huge fan of coverage checkers, but any reason we're explicitly disabling the coverage checker here?

Reply:
Mental lapse? I thought I had set all four of these to `true`, but I guess not...

Reply:
@julz this has been fixed